src/flows/pipes/RawFluxEditor/view.tsx

```diff
@@ -28,8 +28,8 @@ import {VariablesContext} from 'src/flows/context/variables'
 
 // Components
 import SecretsList from 'src/flows/pipes/RawFluxEditor/SecretsList'
-import Functions from 'src/flows/pipes/RawFluxEditor/FunctionsList/GroupedFunctionsList'
-import DynamicFunctions from 'src/flows/pipes/RawFluxEditor/FunctionsList/DynamicFunctionsList'
+import Functions from 'src/shared/components/functions/FunctionsList/GroupedFunctionsList'
+import DynamicFunctions from 'src/shared/components/functions/FunctionsList/DynamicFunctionsList'
```

Comment:
These are already shared components. Moved to shared dir.

Reply:
Existing code was keeping shared components under flows, even though timeMachine also uses it. So it was a move into the shared part...without any further work to de-dupe the components (since outside scope of this PR).
.

Reasoning (which can be changed) in the dir:
* `src/shared/components/functions/`
  * the components used in the function browsers
* subdir `/FunctionsList`
  * our 2 FunctionList styles, specifically for functions.
  * are used conditionally in both timeMachine and flows.
* subdir `/perFunction`
  * reusable tooltip contents, specific for function.

.

We could probably stand to condense these ^^ even further. But at the time, there was much inter-team negotiation to get it even down to this much shared components. 
* As a follow up to this PR, we could try that process of moving them closer again.
* Especially since the additional changes since, have moved these components closer (except not the tooltips).

The question is whether or not we want to holdup this FluxBrowser PR.